application_handlers.go

```diff
@@ -258,16 +258,16 @@ func ApplicationPause(c echo.Context) error {
 	kafkaHeaders := service.ForwadableHeaders(c)
 
 	// Raise the resume event for the source.
```

Comment:
In comment should be probably "Raise the pause event ...."

Reply:
You are right, thank you! Also, it should be "... for the application", not the source :see_no_evil: